### PR TITLE
vercel: add page

### PR DESCRIPTION
**Pull request by TGIFelix**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages/common/vercel.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# vercel
+
+> Vercel CLI provides a set of commands that allow you to deploy and manage your projects.
+> https://vercel.com/docs/cli
```
Comment (bl-ue) [inline pages/common/vercel.md:4]:
```suggestion
> More information: <https://vercel.com/docs/cli>.
```

File: pages/common/vercel.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+- Deploy production:
+
+`vercel --prod`
+
+- Deploy a custom directory:
+
+`vercel /path/to/my_project`
+
+- Remove a deployment
```
Comment (bl-ue) [inline pages/common/vercel.md:18]:
```suggestion
- Remove a deployment:
```

File: pages/common/vercel.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+`vercel`
+
+- Deploy production:
+
+`vercel --prod`
+
+- Deploy a custom directory:
+
+`vercel /path/to/my_project`
```
Comment (bl-ue) [inline pages/common/vercel.md:16]: prefer relative when possible (also simpler :wink:)
```suggestion
`vercel {{path/to/project}}`
```

File: pages/common/vercel.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+
+- Remove a deployment
+
+`vercel rm <project name>`
```
Comment (bl-ue) [inline pages/common/vercel.md:20]: We have specific syntax for user-provided values:
```suggestion
`vercel rm {{project_name}}`
```

**Review by bl-ue — CHANGES_REQUESTED** [review]
Hello @TGIFelix and thank you for the new page! I've left some comments below for you to review.

**Review by bl-ue — CHANGES_REQUESTED** [review]
I've suggested some grammar changes.  Please feel free to discuss them.

**Review by navarroaxel — COMMENTED** [review]
I think we should add examples for `init`, `--build-env`, `--env` and `--regions`.
Looks like we have subcommands here. Also `now` was replaced by `vercel`, so I think we should delete now.md on this PR.

**Review by bl-ue — APPROVED** [review]

File: pages/common/vercel.md [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+
+`vercel {{path/to/project}}`
+
+- Initialize an example project:
+
+`vercel init`
+
+- Deploy with Environment Variables:
+
+`vercel -e {{ENV}}={{var}}`
```
Comment (navarroaxel) [inline pages/common/vercel.md:24]:
```suggestion
`vercel --env {{ENV}}={{var}}`
```

File: pages/common/vercel.md [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+
+- Set default regions to enable the deployment on:
+
+`vercel --regions {{region_id}}`
+
+- Remove a deployment:
+
+`vercel rm {{project_name}}`
```
Comment (navarroaxel) [inline pages/common/vercel.md:32]: Please use long argument format
```suggestion
`vercel remove {{project_name}}`
```

**Review by navarroaxel — CHANGES_REQUESTED** [review]
please add the `--build-env`

**Comment by bl-ue** [comment]
@TGIFelix also delete `now.md`.

**Review by bl-ue — APPROVED** [review]

**Review by sbrl — APPROVED** [review]
Hey there, @TGIFelix! Welcome to tldr-pages. This looks great to me - thanks!

**Review by navarroaxel — APPROVED** [review]

**Comment by TGIFelix** [comment]
Thanks a lot for your help folks. Sorry for such a messy pull request, but at least I know now for future contributions!

**Comment by navarroaxel** [comment]
Don't worry @TGIFelix that was a common review process!
Also `vercel` has a lot of subcommands like `vercel secrets`. You can send more PR's adding pages like `vercel-secrets.md`, `vercel-env.md`, etc.

**Pull request merged**